cmd/environments/get.go

```diff
@@ -76,11 +76,7 @@ func runGet(
 			return errors.NewError(output)
 		}
 
-		output, err := output.CmdOutputSingular(
-			viper.GetString(cliflags.OutputFlag),
-			response,
-			output.SingularPlaintextOutputFn,
-		)
+		output, err := output.CmdOutput("get", viper.GetString(cliflags.OutputFlag), response)
```

Comment:
We need to know if the plaintext output should include a success message, so we pass in this value.